desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -125,12 +164,23 @@ void TabDiveInformation::updateData()
 	updateWhen();
 	ui->waterTemperatureText->setText(get_temperature_string(current_dive->watertemp, true));
 	ui->airTemperatureText->setText(get_temperature_string(current_dive->airtemp, true));
-	updateSalinity();
-
-	ui->atmPressType->setEditable(true);
 	ui->atmPressType->setItemText(1, get_depth_unit());  // Check for changes in depth unit (imperial/metric)
 	ui->atmPressType->setEditable(false);
 	ui->atmPressType->setCurrentIndex(0);  // Set the atmospheric pressure combo box to mbar
+	if (current_dive->salinity) {			// Set water type indicator (EN13319 = 1.020 g/l)
+		if (current_dive->salinity < 10050) 
```

Comment:
Trailing whitespace

Reply:
The previous push did not go as expected. I cannot see white space. Can you still see it? If so then please indicate explicitly